backend/benefit/companies/services.py

```diff
@@ -0,0 +1,28 @@
+from common.utils import update_object
+from companies.models import Company
+
+from shared.ytj.ytj_client import YTJClient
```

Comment:
As we aren't sure yet whether we the YTJClient functionality is sufficient for our purposes, after merging this PR we should investigate also the other options (palveluväylä, other YTJ apis).

Reply:
Sure, this is a temporary solution until we have a decision of which API will be used